pkg/berglas/berglas_test.go

```diff
@@ -49,16 +49,21 @@ func TestGsecretsIntegration(t *testing.T) {
 	}
 	sa = fmt.Sprintf("serviceAccount:%s", sa)
 
-	object := testUUID(t)
+	object, object2 := testUUID(t), testUUID(t)
+	// ensure non-matching prefix
+	for object[:3] == object2[:3] {
```

Comment:
This will panic if our UUID is < 3 characters. While that's technically not possible, it'd be good to put a guard in, since tracing that panic would be annoying. Also we should bound this to something reasonable:

```go
# untested
for i := -; i < 10 && len(object) > 3 && len(object2) > 3 && object[:3] == object2[:3] {
    object2 = testUUID(t)
}
```